clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -3738,6 +3846,17 @@ raw2trace_t::raw2trace_t(
     decode_cache_.reserve(cache_count);
     for (int i = 0; i < cache_count; ++i)
         decode_cache_.emplace_back(cache_count);
+
+#if defined(AARCH64)
+    // TODO i#5365: The decoder uses a global sve_veclen variable to store the vector
+    // length value it uses when decoding. drdecodelib ends up being linked into
+    // drcachesim twice: once into the drcachesim executable, and one into libdynamorio.
```

Comment:
We wanted to avoid needing full standalone DR in raw2trace but were unable to do so.  Given that we have standalone DR, we should remove the drdecode that's linked in.  That should avoid all the duplication.  (Xref #1684 on having libdynamorio link the actual drdecode which I think is not simple; but even w/ that we'd need to change these executable to use static libdynamorio to de-dup with both via that route.). (Xref the missing DR_API which should be fixed as well if these routines are not available in libdynamorio.)